src/pages/HomePage/Homepage.css

```diff
@@ -0,0 +1,105 @@
+.home-page-wrapper {
+  height: 100vh;
+  width: 100%;
+  position: relative;
+  background-position: 100%;
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-size: cover;
+  background-color: #000;
+  color: #1b5a7c;
+  background-image: url("https://cdn.shopify.com/s/files/1/1644/5247/files/1_e5ca8b54-f425-48b7-ba74-edbb51089bd3.jpg?v=1593439585");
+}
+
+.home-page-top {
+  position: absolute;
+  top: 5%;
+  left: 5%;
+}
+
+.home-page-top{
+    color:#ff5a58;
+}
+
+.home-page-title {
+  color: #fbbc04;
+}
+
+.home-content-container {
+  position: absolute;
+  top: 20%;
+  left: 5%;
+}
+
+.home-content {
+  display: flex;
+  align-items: center;
+  margin: 2rem 0rem 1rem 0rem;
+  color: #fbbc04;
+}
+.home-content span {
+  display: block;
+  width: 5%;
+  border-top: 2px solid #ccc;
+}
+.home-content h4 {
+  margin-left: 1rem;
+}
+.home-content-description {
+  width: 40%;
+}
+.home-content-description h1 {
+  font-size: 2rem;
+  font-family: sans-serif;
```

Comment:
`font-family` you can define under universal selector (*)

Reply:
But for the other components I will use a different font family